src/Slices/ResourceDiscovery/UI/DiscoveredResourcesRow.tsx

```diff
@@ -40,37 +38,52 @@ export const DiscoveredResourceRow: React.FC<Props> = ({
         </Td>
         <Td
           dataLabel={words("discovered.column.resource_id")}
-          style={{ width: "50vw" }}
           data-testid={words("discovered.column.resource_id")}
         >
-          {row.discovered_resource_id}
+          <Truncate content={row.discovered_resource_id} />
```

Comment:
This can have unexpected behaviours I've noticed.
You should be able to add the `modifier="truncate"` on the TD element like so:
`<Td modifier="truncate">content</Td>` 
That will respect the width set by the Th element if it's set for example.